src/core/ext/filters/client_channel/subchannel.cc

```diff
@@ -676,7 +676,7 @@ Subchannel::Subchannel(SubchannelKey key,
             {GRPC_MAX_CHANNEL_TRACE_EVENT_MEMORY_PER_NODE_DEFAULT, 0,
              INT_MAX}));
     channelz_node_ = MakeRefCounted<channelz::SubchannelNode>(
-        grpc_sockaddr_to_uri(&key_.address()).value(),
+        grpc_sockaddr_to_uri(&key_.address()).value_or(""),
```

Comment:
The empty string isn't very useful.  Let's make it something like "<unknown address type>".

Reply:
Looks like github munged my comment.  I meant something like `<unknown address type>`.